src/main/java/com/sorting/Selection.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * File: Selection.java
+ * Date: 10 Dec, 2015
+ * Author: Tarun Jain
+ * -----------------------------------------------
+ * Sorting is the process of rearranging a sequence of objects so as to put them 
+ * in some logical order. The selection sort algorithm sorts an array by repeatedly 
+ * finding the minimum element (considering ascending order) from unsorted part and 
+ * putting it at the beginning. It exchange one for every pass through the list. 
```

Comment:
Description is correct. But i have heard in some video very simple logic of selection sort. Once i revisit it i will add that explanation too.